### ISSUE TITLE
Unexpected token / in JSON at position 59

### ISSUE THREAD

**Issue by acoard** [SimonTestNet/SimonTest#21]
I used SimonTest in the past (weeks ago), but now it's unable to work at all for me.  I reduced the component down to it's bare bones, and I still get the same error.

```typescript
import {ChangeDetectorRef, Component, Injectable, ViewChild} from '@angular/core';

@Component({
    templateUrl: './documents.component.html'
})
@Injectable()
export class AccountDocumenTestComponent  {
}
```

The line number in the error never changes from 59, even when adding/removing lines.

Running SimonTest 0.17.9.

**Comment by ManuelDeLeon** [comment]
I can generate a test for that block of code no problem. Could it be some kind of encoding or weird invisible character? A few things to try:

- Copy/paste that block from here on github to a new file and generate the tests.
- Create a new project and try there.
- Reinstall SimonTest

**Comment by ManuelDeLeon** [comment]
See if you get the same error as in #20

**Comment by ManuelDeLeon** [comment]
I think it's an issue with package.json or tslint.json (where the formatting rules are read). Can you attach both files?

**Comment by kpaxton** [comment]
I had this issue as well. I noticed that I had put comments in the tslint.json file. Once I removed them it worked again.

**Comment by ManuelDeLeon** [comment]
This is fixed on v0.18.1
Thanks @kpaxton
